testsuite/MDAnalysisTests/test_tprparser.py

```diff
@@ -246,8 +246,8 @@ def _test_is_in_topology(name, elements, topology_section, topology_path):
     """
     universe = MDAnalysis.Universe(topology_path)
     for element in elements:
-        assert element in universe._topology[topology_section], \
-            'Interaction type "{0}" not found'.format(name)
+        if element not in universe._topology[topology_section]:
+            raise Exception('Interaction type "{0}" not found'.format(name))
```

Comment:
or even better use the `assert_` function. This assert isn't optimized away and the code still reads the same.